Scripts/AnalyzeDatabank/NMRPCA_timerelax.py

```diff
@@ -23,19 +23,20 @@
 from DatabankLib.analyze import computeNMRPCA
 
 
-systems = initialize_databank()
-resDict = {DatabankLib.RCODE_COMPUTED: 0, 
-           DatabankLib.RCODE_SKIPPED: 0,
-           DatabankLib.RCODE_ERROR: 0}
-
-for system in systems:
-    logger.info("System title: " + system['SYSTEM'])
-    logger.info("System path: " + system['path'])
-    res = computeNMRPCA(system)
-    resDict[res] += 1
-
-print(f"""
-COMPUTED: {resDict[DatabankLib.RCODE_COMPUTED]}
- SKIPPED: {resDict[DatabankLib.RCODE_SKIPPED]}
-   ERROR: {resDict[DatabankLib.RCODE_ERROR]}
-""")
\ No newline at end of file
+if __name__ == "__main__":
```

Comment:
these can stay like this, but in the future we should wrap all these into their own methods and call them in the main

Reply:
I made a runner in utils.py and now runner-scripts are just wrappers as @pbuslaev adviced.
See 050d0dfe